lib/settings.h

```diff
@@ -191,8 +191,8 @@ class CPPCHECKLIB Settings : public cppcheck::Platform {
         Default value is 0. */
     int exitCode;
 
-    /** @brief --file-filter for analyzing special files */
-    std::string fileFilter;
+    /** @brief List of --file-filter for analyzing special files */
+    std::list<std::string> fileFilters;
```

Comment:
I have read somewhere that vector should be the default container.. unless there is a good reason to use something else.

Reply:
You might have read that in "Effective STL" :-D 
Yeah vector build upon on arrays and are therefore cheap and support random access. List afaik are linked and therefore have some overhead, but support cheap inserting and erasing from anywhere in the list.

There is no performance reason or anything like that I chose std::list over std::vector here, it's just that the source codes already uses lists in the places where I looked and I therefore used it for consistency. 
But I also saw now that std::vector is used too, so I will change it